src/modules/sensors/sensors.cpp

```diff
@@ -1542,6 +1506,39 @@ Sensors::parameter_update_poll(bool forced)
 					(void)sprintf(str, "CAL_MAG%u_ZSCALE", i);
```

Comment:
It was, but I do realize that it can look odd. I've changed to the more verbose logical or style, thanks!